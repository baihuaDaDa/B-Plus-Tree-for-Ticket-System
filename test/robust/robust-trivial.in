39
insert a 10
insert a 20
insert a 30
insert a 40
insert a 50
insert a 60
insert a 70
insert a 80
insert a 43
insert a 47
insert a 83
insert a 87
find a
delete a 10
delete a 20
delete a 30
delete a 40
delete a 50
delete a 60
delete a 70
delete a 80
delete a 43
delete a 47
delete a 83
delete a 87
find a
insert a 10
insert a 20
insert a 30
insert a 40
insert a 50
insert a 60
insert a 70
insert a 80
insert a 43
insert a 47
insert a 83
insert a 87
find a