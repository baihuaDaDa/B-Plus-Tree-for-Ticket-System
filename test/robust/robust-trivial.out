10 20 30 40 43 47 50 60 70 80 83 87 
null
10 20 30 40 43 47 50 60 70 80 83 87 
